src/cozmo/conn.py

```diff
@@ -73,7 +73,7 @@ class EvtConnectionClosed(event.Event):
 # Some messages have no robotID but should still be forwarded to the primary robot
 FORCED_ROBOT_MESSAGES = {"AnimationAborted",
                          "AnimationEvent",
-                         "AvailableObjects",
+                         "ObjectStates",
```

Comment:
Minor, but these were in alphabetical order to make it easier to spot if a message was included, please move it a few lines down.